Extensions/Today/Images.xcassets/widgetLabelColors.colorset/Contents.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Here as well we can get rid of colorset since you are using TodayUX.labelColor and subtitleLabelColor